volontulo/tests/views/test_organizations.py

```diff
@@ -165,8 +169,222 @@ def test__create_valid_organization_form_post(self):
+        self.assertContains(response, u'Edytuj organizację')
+        self.assertContains(response, u'Dodaj ofertę')
+
+        self.client.post('/login', {
+            'email': u'organization2@example.com',
+            'password': 'organization2',
+        })
+        response = self.client.get(
+            '/organizations/organization-1/1',
+            follow=True
+        )
+        self.assertNotContains(response, u'Edytuj organizację')
+        self.assertNotContains(response, u'Dodaj ofertę')
+        self.assertIn('allow_contact', response.context)
+        self.assertTrue(response.context['allow_contact'])
+        self.assertFalse(response.context['allow_edit'])
+        self.assertFalse(response.context['allow_offer_create'])
+
+    def test__get_filled_organization_view_by_organization(self):
+        u"""Request for filled organization view by organization user"""
+        self.client.post('/login', {
+            'email': u'organization2@example.com',
+            'password': 'organization2',
+        })
+        response = self.client.get(
+            '/organizations/organization-2/2',
+            follow=True
+        )
+        # pylint: disable=no-member
+        self.assertIn('offers', response.context)
+        self.assertEqual(len(response.context['offers']), 4)
+
+    def test__post_contact_form_on_organization_view_by_anonymous(self):
+        u"""Post contact form to organization view by anonymous user"""
+        form_params = {
+            'name': u'',
+            'email': u'',
+            'phone_no': u'',
+            'message': u'',
+        }
+        response = self.client.post(
+            '/organizations/organization-1/1',
+            form_params,
+            follow=True
+        )
+
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, u"Formularz zawiera nieprawidłowe dane")
+
+        form_params = {
+            'name': u'Mister Volunteer',
+            'email': u'mister.volunteer@example.com',
+            'phone_no': u'+48 123 123 123',
+            'message': u'',
+        }
+        response = self.client.post(
+            '/organizations/organization-1/1',
+            form_params,
+            follow=True
+        )
+
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, u'Formularz zawiera nieprawidłowe dane')
+        self.assertContains(response, u'Mister Volunteer')
+        self.assertContains(response, u'+48 123 123 123')
+        self.assertContains(response, u'mister.volunteer@example.com')
+
+        form_params = {
+            'name': u'',
+            'email': u'',
+            'phone_no': u'',
+            'message': u'Wiadomość dla organizacji',
+        }
+        response = self.client.post(
+            '/organizations/organization-1/1',
+            form_params,
+            follow=True
+        )
+        self.assertContains(response, u'Wiadomość dla organizacji')
```

Comment:
I have no idea, if in this test conatact form has been sent successfully or not. I believe test should explain that.